thunder/examine/memory_caculation.py

```diff
@@ -1,12 +1,15 @@
-from collections.abc import Callable
+from collections.abc import Callable, MutableSequence, MutableMapping, MutableSet
+from functools import partial
 
 from thunder.core.prims import PrimIDs
-from thunder.core.proxies import FutureTensorProxy, Proxy, TensorProxy
+from thunder.core.proxies import CollectionProxy, FutureTensorProxy, Proxy, TensorProxy
 from thunder.core.symbol import BoundSymbol, Symbol
 from thunder.core.trace import TraceCtx
 from thunder.core.utils import check_type, ProxyDict
+from thunder.core.pytree import tree_iter
 
-memory_calculate_skip_list = (PrimIDs.RETURN, "clear_collection")
```

Comment:
`"clear_collection"` has changed its name to `"clear_mutable_collection"`, which should not be skipped.

Reply:
Thanks, updated in 136c7d16cf1a963279101272c8728517a9966b60